include/alpaka/mem/buf/uniformCudaHip/Set.hpp

```diff
@@ -84,20 +85,25 @@ namespace alpaka
                     Dim<TView>::value == Dim<TExtent>::value,
                     "The destination buffer and the extent are required to have the same dimensionality!");
 
-                ALPAKA_UNIFORM_CUDA_HIP_RT_CHECK(ALPAKA_API_PREFIX(MemsetAsync)(
-                    getPtrNative(this->m_view),
+                auto& view = this->m_view;
+                ALPAKA_ASSERT(1 <= getWidth(view));
```

Comment:
Why did you readd this check? The `view` is zero dimensional, it must have a width of 1.